packages/components/src/components/hds/application-state/body.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export interface HdsApplicationStateBodySignature {
+  Args: {
+    text?: string;
+  };
+  Blocks: {
+    default?: [];
+  };
+  Element: HTMLDivElement;
+}
+
+export default class HdsApplicationStateBodyComponent extends Component<HdsApplicationStateBodySignature> {}
+
+declare module '@glint/environment-ember-loose/registry' {
+  export default interface Registry {
+    'Hds::ApplicationState::Body': typeof HdsApplicationStateBodyComponent;
+    'hds/application-state/body': typeof HdsApplicationStateBodyComponent;
+  }
+}
```

Comment:
Thanks for addressing the feedback! We have a shared registry (`template-registry.ts`) where we export the aliases. If you've not seen this yet, we have [a short migration guide](https://github.com/hashicorp/design-system/blob/main/wiki/TypeScript-Migration.md) to make sure we keep things consistent. We'd also need a re-export in `packages/components/src/components.ts` (see step 7).